includes/blocks/podcast/markup.php

```diff
@@ -46,8 +47,26 @@
 	$term_image_id = '';
 }
 
+/**
```

Comment:
Nitpick: Only docblocks need the double asterisk, multi line comments do not.

```suggestion
/*
```